locopy/utility.py

```diff
@@ -317,6 +318,20 @@ def validate_float_object(column):
         except (ValueError, TypeError):
             return None
 
+    def check_column_type_pyarrow(pa_dtype):
+        if pa.types.is_temporal(pa_dtype):
+            return "timestamp"
+        elif pa.types.is_boolean(pa_dtype):
+            return "boolean"
+        elif pa.types.is_integer(pa_dtype):
+            return "int"
+        elif pa.types.is_floating(pa_dtype):
+            return "float"
+        elif pa.types.is_string(pa_dtype):
+            return "varchar"
+        else:
+            return None
```

Comment:
Should the default be varchar? Just looking at the regular Pandas dtype if/else block. Maybe I misread.

Reply:
yea you are right. I just didn't think it was nice to have that as a default (sounds like a silent error). On a second thought, I think we should keep varchar as  default

Reply:
Ya, just want to be consistent